src/Umbraco.Web.UI.Client/src/packages/core/sorter/property/actions/sort-mode/manifests.ts

```diff
@@ -0,0 +1,24 @@
+import type { UmbExtensionManifestKind } from '@umbraco-cms/backoffice/extension-registry';
+import { UMB_PROPERTY_ACTION_DEFAULT_KIND_MANIFEST } from '@umbraco-cms/backoffice/property-action';
+
+export const UMB_PROPERTY_ACTION_SORT_MODE_KIND_MANIFEST: UmbExtensionManifestKind = {
```

Comment:
I would not put this sort-mode property action inside the `sorter` package. It feels to me like two completely different things. I understand they name wise are close, but I think its best if they are kept apart as they code wise are not connected.

Maybe its own package would be a good call? called `sort-mode`? Let me know what you think